### PR TITLE
Pd 368 px essentials

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

**Comment by nathan-px** [comment]
@jrivera-px @prashantsr I created a supported feature matrix and a brief paragraph describing PX-Essential. I think that's all that we provided for in the current doc spec. Is there anything more that we want/need to do for this?

**Comment by prashantsr** [comment]
I think we need to change a few things:
- give link to website (when available) so that they register from website (for marketing) and then that redirects them to central.
- I think product/feature description is fine. We may have to fine tune the language but that is all it is.
- we should explain that only one cluster can be registered per user and the activation process (license is embedded and requires connection to PX-Central for activation and ongoing operation)

**Comment by nathan-px** [comment]
> I think we need to change a few things:
> 
> * give link to website (when available) so that they register from website (for marketing) and then that redirects them to central.
> * I think product/feature description is fine. We may have to fine tune the language but that is all it is.
> * we should explain that only one cluster can be registered per user and the activation process (license is embedded and requires connection to PX-Central for activation and ongoing operation)

Ok, I just pushed an update on your comments. We can change the central.portworx.com link when a more accurate one is available.

File: content/concepts/portworx-essentials/_index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+To use Portworx Essentials, you must activate your license online and your cluster must be connected to PX-Central using the internet. Get your free PX-Essentials license when you sign up on [PX-Central](https://central.portworx.com/).
+
+## Portworx Essentials available features and limitations
+
+The following tables list what features Portworx Essentials supports and any limitations it may have:
+
+### Portworx Essentials
+
+| **Feature** | **Support** |
```
Comment (nathan-px) [inline content/concepts/portworx-essentials/_index.md:20]: @prashanthpx These tables are different from the table on the marketing page. I'll try to paste in direct matches,  but it's not 1-to-1.

**Pull request merged**